content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,130 @@
+
+You have a few options with DVC for this.
+
+- Set up an external cache on your object storage bucket.
+  [Check out this doc to learn more](https://dvc.org/doc/user-guide/managing-external-data#setting-up-an-external-cache)!
+- Track a large dataset in an external location to download later.
+  [This doc will tell you more](https://dvc.org/doc/command-reference/add#example-transfer-to-remote-storage).
+
+Great question @Ravi Kumar!
+
+### [Q: Is there a recommended way to specify multiple pipelines in DVC?](https://discord.com/channels/485586884165107732/485596304961962003/864230750325047316)
+
+You'll want to keep each pipeline in a separate `dvc.yaml` if you want to work
+with multiple pipelines. Here's a bit of explanation:
+
+- Splitting a `dvc.yaml` file into multiple files is encouraged where there are
+  clear logical groupings between stages. It avoids confusion, improves
+  readability, and shortens commands by avoiding long paths preceding every
+  filename.
+- `dvc.yaml` files can be in any sub-directory or nested sub-directory in the
+  project structure and DVC will find them.
+- DVC will process them just the same as if they were one DVC file i.e.
+  dependencies between stages in different `dvc.yaml` files are still respected.
+- Each `dvc.yaml` file will have its own `dvc.lock` file in the same directory.
+
+If you want to see the rest of the explanation,
+[check out this user guide PR we have up](https://github.com/iterative/dvc.org/issues/2494).
+Please feel free to add a comment or emoji on this PR so we know how to
+prioritize this content for you!
+
+Thanks @Tups!
+
+### [Q: Is there way to allow different pipelines to have common dependencies and outputs in DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/867747202306146335)
+
+Good question @vgodie!
+
+It is possible to have overlapping dependencies, but not overlapping outputs.
+Having overlapping outputs introduces uncertainty into DVC commands, like
+`dvc checkout`.
+
+Sometimes people want to have overlapping directory outputs (different stages
+that wrote many different files in the same directory). They might have a series
+of stages that append to the same file. In this case, we suggest creating new
+files and combining them in a final stage so they are consistently written in
+the same order.
+
+### [Q: How does the CML runner restart workflows if it's been shut down by AWS (e.g. spot instances)?](https://discord.com/channels/485586884165107732/728693131557732403/862641924200857660)
+
+You shouldn't have to do anything. Spot instances sends a `SIGINT` that we
+handle to restart the workflow. We have been supporting graceful shutdown by
+unregistering runners for a while now.
+
+The main difference now is that we restart workflows with unfinished jobs.
+
+We know for sure this isn't working in Azure. Azure does not do a graceful
```

Comment:
do we have a workaround? ticket? 

or consider removing this, since question is about AWS

Reply:
I removed it because the question was about AWS.